### ISSUE TITLE
Simple XML/JSON export 

### ISSUE THREAD

**Issue by TheAdventist** [khuttun/NotificationNotes#30]
I was wondering if it is possible to include an export function which allows you to save an XML/JSOn (or whatever you use to store the notes) and push it as an download at the users request.
This is in particular usefull if you want to save your notes (either just to have it or when migrating to a new phone).

Or perhaps an link to the location of the notes/xml (in the File Manager) so users can decide themself what to do with the data...

**Comment by khuttun** [comment]
Good idea. It should be checked whether this could be combined somehow with #25 .
